ia/test/unitTest.py

```
@@ -0,0 +1,29 @@
+import unittest
+import sys
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parent.parent / 'src'))
+
+import message as msg
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)